src/ray/raylet/worker_pool.h

```diff
@@ -129,6 +128,14 @@ class WorkerPool {
   /// Record metrics.
   void RecordMetrics() const;
 
+  /// Tick the heartbeat timer and get the workers that have timed out.
+  ///
+  /// \param[in] max_missed_heartbeats The maximum number of heartbeats that can be
+  /// missed before a worker times out.
+  /// \param[out] dead_workers Workers that have been dead.
```

Comment:
Does this mean workers that were already dead, all workers that have ever died or those that have died since the last time this was called? Please make the comment more specific.